flow/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ target_link_libraries(flowlinktest PRIVATE flow stacktrace)
 find_package(ZLIB)
 if(ZLIB_FOUND)
   add_compile_definitions(ZLIB_LIB_SUPPORTED)
```

Comment:
I think I prefer `target_compile_definitions(flow PUBLIC ZLIB_LIB_SUPPORTED)` so that this only affects targets depending on flow instead of all targets